tracing-test-utils/src/main/java/com/palantir/tracing/TestTracingExtension.java

```diff
@@ -32,18 +32,23 @@
 import org.junit.jupiter.api.extension.AfterTestExecutionCallback;
 import org.junit.jupiter.api.extension.BeforeTestExecutionCallback;
 import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ExtensionContext.Namespace;
 import org.junit.platform.commons.support.AnnotationSupport;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 final class TestTracingExtension implements BeforeTestExecutionCallback, AfterTestExecutionCallback {
 
     private static final Logger log = LoggerFactory.getLogger(TestTracingExtension.class);
-    private final TestTracingSubscriber subscriber = new TestTracingSubscriber();
+    private static final Namespace TRACING_EXT_NAMESPACE = Namespace.create(TestTracingExtension.class);
```

Comment:
nit: This is in the `TestTracingExtension` class, so the qualifier is not necessary.

```suggestion

    private static final Namespace NAMESPACE = Namespace.create(TestTracingExtension.class);
```